### DISCUSSION TITLE
Tidy up html classes in erb

### DISCUSSION THREAD

**Discussion by andyhennie** [rubocop/rubocop#9379] [Q&A]
Is it possible to configure RuboCop so it sorts/tidy up the css classes in your html alphabetically? Eg. from `<div class="px-1 my-1 w-full">` to `<div class="my-1 px-1 w-full">` ?

**Comment by dvandersluis** [comment]
I'm afraid RuboCop cannot do this. RuboCop parses ruby code, not CSS or HTML. Perhaps https://github.com/stylelint/stylelint or https://github.com/CSSLint/csslint can do what you're looking for but we can't vouch for those projects.

**Comment by koic** [comment]
I totally agree with @dvandersluis. I think that inline Ruby code of template engines like ERB and Haml will also not supported by RuboCop core. They are implemented by third party linters. e.g. https://github.com/Shopify/erb-lint, https://github.com/sds/haml-lint.
